txpool/account.go

```diff
@@ -19,20 +19,15 @@ type accountsMap struct {
 
 // Intializes an account for the given address.
 func (m *accountsMap) initOnce(addr types.Address, nonce uint64) *account {
-	a, _ := m.LoadOrStore(addr, &account{})
+	a, _ := m.LoadOrStore(addr, &account{
```

Comment:
Just to increment count, you can check 2nd returns of `LoadOrStore` instead of using `init.Do`

Reply:
Done https://github.com/0xPolygon/polygon-edge/pull/1104/commits/3f2cba1bf1a10aa113cf4647ee9714a93d1b7754